packages/TextEditor/src/types.js

```diff
@@ -0,0 +1,4 @@
+export const kind = {
+  SIMPLE: "simple",
+  ADVANCE: "advance",
```

Comment:
Sorry, this one too 😊 
```suggestion
  ADVANCED: "advanced",
```

Reply:
(need the grammar to be right from the beginning)